CHANGELOG.md

```diff
@@ -4,6 +4,37 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [Unreleased]
```

Comment:
Since this introduces major breaking changes, should we copy the *previous* README.md file to the .gihub folder such that the 0.10.0 README will be shown on github? See #314

Reply:
You mean for the phase where the PR is merged in main but 0.11.0 is not yet released? What would our strategy then look like, would we update this automatically in our release pipeline somehow?

Reply:
Exactly. We could do this step every time when having a PR that creates the new release version, this one now is basically just the "delayed" version as 0.10.0 happened during my OOO time and I thus could not discuss this with you. I also thought about automating this, but I did not find a nice way of doing so.

Reply:
I'll close this issue here because I urgently want the PR to be completed and the topic is a more general one. However, I'm happy to discuss this any further on one of the doc PRs if you want